Source/Render/sokol/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ IF (OPTION_PROCESS_SHADERS)
 
     if(NOT DEFINED ENV{PERIMETER_SHADER_LANG})
         #Use all GLSL dialects, HLSL for D3d and WebGPU
-        set(PERIMETER_SHADER_LANG "glsl330:glsl300es:hlsl5:wgsl")
```

Comment:
`wsgl` does not support `sampler2DShadow` but in Sokol examples they use Open GL ES for this